src/appkit.cs

```diff
@@ -15099,6 +15172,15 @@ interface NSStoryboard {
 		[Static]
 		[NullAllowed, Export ("mainStoryboard", ArgumentSemantic.Strong)]
 		NSStoryboard MainStoryboard { get; }
+
+		[Mac (10,15)]
+		[Export ("instantiateInitialControllerWithCreator:")]
+		[return: NullAllowed]
+		NSObject InstantiateInitialController ([NullAllowed] NSStoryboardControllerCreator block);
+
+		[Mac (10,15)]
+		[Export ("instantiateControllerWithIdentifier:creator:")]
+		NSObject InstantiateController (string identifier, [NullAllowed] NSStoryboardControllerCreator block);
```

Comment:
we prefer handler over block @dalexsoto dixit.
```suggestion
		NSObject InstantiateController (string identifier, [NullAllowed] NSStoryboardControllerCreator handler);
```